agent/instrumentation/micrometer-1.0/src/test/groovy/MicrometerTest.groovy

```diff
@@ -226,6 +226,6 @@ class MicrometerTest extends AgentInstrumentationSpecification {
     def measurements = delegate.getMeasurements().stream()
       .filter({ it.name == name && it.value != 0 })
       .collect(Collectors.toList())
-    return measurements.get(measurements.size() - 1)
+    return measurements.size() > 0 ? measurements.get(measurements.size() - 1) : null;
```

Comment:
I think returning `null` will end up throwing NullPointerException above on line 219

Reply:
will revert it now after skipping cache successfully.

Reply:
or adding null check everywhere?